pyanaconda/dbus/__init__.py

```diff
@@ -47,25 +48,30 @@ def get_connection():
         return DBus._connection
 
     @staticmethod
-    def get_new_connection():
+    def get_new_connection(bus_address=None):
```

Comment:
I don't think that this argument should be part this method, because it is not clear how it should be used. 

Since the only way how to use the method with the argument is this:
`DBus._connection = DBus.get_new_connection(TEST_ADDRESS)`

It doesn't look any better then this:
`DBus._connection = pydbus.connect(bus_address)`

I would prefer not to add this confusing option just for testing.

Btw. the best way is to set up the `DBUS_STARTER_ADDRESS` anyway.

Reply:
Setting environment variable is not the easy way and it could be really unpleasant to work with. Also I don't see the confusion, you could specifying address for DBus?

Reply:
Let's discuss it tomorrow.